fix: #234 add margin to prevent input field overlap in password reset…

Sanchez86 opened the pull request:
## Description
Fixed input field overlap issue in the password reset form by adding margin-bottom spacing.

## Changes
- Added `sx={{ mb: 2 }}` to Box wrapper around TitleWithDescription component
- This prevents the title from overlapping with the email input field

## Issue
Resolves task #234

Sanchez86 committed:
fix: #229 Add scroll to top functionality on logo click

- Add handleLogoClick function to NavBar component
- Implement smooth scroll to top when logo is clicked
- Add main-content ID to AppMain container for reliable targeting
- Navigate to home page if not already there
- Replaces unreliable CSS class selector with stable ID selector
- Add comprehensive test coverage for handleLogoClick function
- Mock document.querySelector and scrollTo methods for testing
- Verify scroll behavior and navigation functionality
- Ensure proper cleanup of mocked functions

Fixes #229

alisa-korniienko committed:
feat(frontend): Google OAuth - use response.credential; POST /auth/google-auth (refs #240, depends on backend PR #38)

Sanchez86 committed:
fix: #234 add margin to prevent input field overlap in password reset form

AndiKot commented on src/containers/guest-home-page/forgot-password/ForgotPassword.jsx:
```diff
@@ -78,7 +78,7 @@ const ForgotPassword = () => {
   return (
     <Box sx={styles.root}>
-      <Box>
+      <Box sx={{ mb: 2 }}>
```
The margin should be 32 px accordingly to the layout. It is 16px for now.
<img width="1016" height="876" alt="image" src="https://github.com/user-attachments/assets/275ac096-6638-496c-86b0-7d86daa12e26" />

Sanchez86 replied:
Already fixed it. Thanks

KorolyovKN committed:
fix-#36-set up sonarcloud ci

Yuliia-Chepak committed:
Fix: update guest-home-page.json translation only

Sanchez86 committed:
Fix margin bottom to the correct value of 32 px

Yuliia-Chepak committed:
Revert "Bugfix/230 home page how it works send request"

KorolyovKN committed:
add docker test

Yuliia-Chepak committed:
[BUGFIX #237] Update 'generalInfo' page description in Become a student pop-up (#13)

KorolyovKN committed:
fix/36 - add docker file to exclisions

KorolyovKN committed:
fix/36 - fix sonar exclisions

KorolyovKN committed:
fix/36 - add full SHA for setup-buildx-action

kriLEBED committed:
test: #55 add test for search filter input component (#18)

* test: #55 add test for search filter input component

Add unit tests for SearchFilterInput with beforeEach setup

Test cases included:
- it should render component with input in it
- it should render typed text correctly
- it should delete typed text when delete button is clicked
- it should call updateFilter function on search button click
- it should call updateFilter function when enter is pressed

* fix prettier errors

PliutaNastya committed:
fix: #236 change become a student pop up the placeholder text in the interests page (en and ua version) (#23)

PliutaNastya committed:
test: #49 add test for content switcher component (#19)

* test: #49 add test for content switcher component

* update: test #49 move render and userEvent to beforeEach

* test

PliutaNastya committed:
test: #51 add test for enhanced table pagination component (#17)

PliutaNastya committed:
test: #48 add test for chips list component (#15)

kriLEBED committed:
test: #56 Write tests for slider with input component (#16)

* test: #56 add test for slider with input component

Add unit tests for  SliderWithInput with beforeEach setup

Test cases included:
- it should renders correctly
- it should call onChange when slider is moved
- it should update inputValue correctly when input value is empty
- it should not update prices when input is blurred and value in input has not changed
- it should update prices when input is blurred and input is greater than max value

Sanchez86 committed:
feat: remove hash from URL after scrolling to top on logo click

- Added hash removal functionality when scrolling to top
- Uses navigate with replace: true to avoid adding history entry
- Improves UX by cleaning URL after returning to page top

Sanchez86 committed:
test: add test coverage for hash removal functionality

- Added test for hash removal from URL when scrolling to top
- Updated NavBar tests with proper mocking for navigate function
- Improved test coverage to 94.36% statements for NavBar component
- Tests now properly verify both scroll behavior and hash removal
